content/blogs/release-0-23.md

```diff
@@ -0,0 +1,711 @@
+
+```yaml
+id: python
+namespace: company.team
+
+tasks:
+  - id: python
+    type: io.kestra.plugin.scripts.python.Script
+    containerImage: python:3.13-slim
+    taskRunner:
+      type: io.kestra.plugin.scripts.runner.docker.Docker
+    dependencies:
+      - pandas
+      - kestra
+    script: |
+      from kestra import Kestra
+      import pandas as pd
+      data = {
+        'Name': ['Alice', 'Bob', 'Charlie'],
+        'Age': [25, 30, 35]
+      }
+      df = pd.DataFrame(data)
+      print(df)
+      print("Average age:", df['Age'].mean())
+      Kestra.outputs({"average_age": df['Age'].mean()})
+```
+
+A new `dependencyCacheEnabled` flag (boolean) allows you to enable or disable caching in the worker directory, so dependencies can be quickly retrieved the next time the task runs. The `beforeCommands` property is still supported for advanced use cases or custom installation steps.
+
+<div class="video-container">
+    <iframe src="https://www.youtube.com/embed/g9Jt5zt9wI4" title="Python Dependency Caching" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe>
+</div>
+
+
+### Git Sync for Apps & Dashboards
+
+Kestra 0.23.0 introduces Git integration for Dashboards and Apps, enabling version control and collaborative management of these resources through familiar Git workflows. You can now:
+
+- **Version control your dashboards and apps** using `git.SyncDashboard`, `git.PushDashboard`, `git.SyncApps`, and `git.PushApps` tasks
+- **Track configuration changes** over time, managing all your resources as code.
+- **Collaborate** with team members using familiar Git workflows
+- **Roll back** to previous versions when needed.
+
+::collapse{title="Example of Pulling Apps from Git to Kestra"}
+
+The following flow allows to pull an Apps from a GitHub repository and deploy it to the Kestra instance:
+
+```yaml
+id: sync_apps_from_git
+namespace: system
+tasks:
+  - id: git
+    type: io.kestra.plugin.ee.git.SyncApps
+    delete: true # optional; by default, it's set to false to avoid destructive behavior
+    url: https://github.com/kestra-io/apps # required
+    branch: main
+    username: "{{ secret('GITHUB_USERNAME') }}"
+    password: "{{ secret('GITHUB_ACCESS_TOKEN') }}"
+triggers:
+  - id: every_full_hour
+    type: io.kestra.plugin.core.trigger.Schedule
+    cron: "0 * * * *"
+```
+::
+
+
+## Notable Enhancements
+
+**Ion data format support** with new `IonToParquet` and `IonToAvro` tasks for data conversion, plus `InferAvroSchemaFromIon` for schema generation.
+
+**Pause Task**: The Pause task now uses a `pauseDuration` property, replacing `delay` and removing `timeout`. When the duration ends, the task proceeds based on the `behavior` property: `RESUME` (default), `WARN`, `FAIL`, or `CANCEL`. Manually resumed tasks always succeed.
```

Comment:
`timeout` is removed because it exist in the parent task, there is no more specific handling of it but the task could still timeout.

There is also a new `onPause` property to define a task that should run when the flow enter a pause.